### PR TITLE
Mobile-view: paragraph section fixes

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
- Fix image path for base spin field icons
- Do not hide or show elements which are force to be hidden in mobile view

### CODE REVIEW

File: browser/src/control/Control.MobileWizardWindow.js [hunk 1]
```diff
@@ -299,13 +299,21 @@ L.Control.MobileWizardWindow = L.Control.extend({
 				this._customTitle ? this._setCustomTitle(this._customTitle) : this._setTitle(this._mainTitle);
 
 			var currentNode = $('.ui-explorable-entry.level-' + this._currentDepth + '.mobile-wizard:visible');
-			var headers = currentNode.siblings();
+			// select only those nodes which are updated on Level down
+			var headers = currentNode.siblings().filter(function() {
+				return $(this).attr('style') === 'display: none;';
```
Comment (pedropintosilva) [inline browser/src/control/Control.MobileWizardWindow.js:304]: is this ok

File: browser/src/control/Control.MobileWizardWindow.js [hunk 2]
```diff
@@ -299,13 +299,21 @@ L.Control.MobileWizardWindow = L.Control.extend({
...
 			var currentHeader = currentNode.children('.ui-header');
 			headers = headers.add(currentHeader);
 
 			var parent = currentNode.parent();
 			if (parent.hasClass('toolbox'))
-				headers = headers.add(parent.siblings());
+				parent.siblings().each(function() {
+					// select only those nodes which are updated on Level down
+					if ($(this).attr('style') === 'display: none;') {
+						headers = headers.add($(this));
```
Comment (pedropintosilva) [inline browser/src/control/Control.MobileWizardWindow.js:314]: Is this ok? Meaning, will it always work? wouldn't be better to `$(this).style.display === 'none'`

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.MobileWizardWindow.js:314]: `$(this).style.display === 'none'` will not work as it consider all the elements which have display = none ( including the elements which we forcefully hide by css )

this condition will only consider elements which have inline property display = none

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.MobileWizardWindow.js:314]: As i am seeing you are right if in future there are addition inline css line added then this fail 🤔

**Comment by pedropintosilva** [comment]
image
 looks good, I need to check the second one, please backport to 23.05 so CI can start

**Review by pedropintosilva — APPROVED** [review]
Looks good but it would be nice to wait and see if https://github.com/CollaboraOnline/online/pull/8517 passes the CI also could you give an example of 

> Do not hide or show elements which are force to be hidden in mobile view

**Comment by Darshan-upadhyay1110** [comment]
> example of Do not hide or show elements which are force to be hidden in mobile view

/* Paragraph properties */
 #paraspacingimg,
 #aboveparaspacingimg,
 #belowparaspacingimg,
 #firstlineindentimg,
 #indentimg,
 #beforetextindentimg,
 #aftertextindentimg
 
 https://github.com/CollaboraOnline/online/blob/master/browser/css/mobilewizard.css#L1234

**Review by eszkadev — APPROVED** [review]
looks ok

**Pull request merged**